tests/components/cast/test_media_player.py

```diff
@@ -7,6 +7,7 @@
 
 import attr
 import pytest
+from pychromecast.socket_client import CONNECTION_STATUS_CONNECTED
```

Comment:
This is not OK, how can I fake `pychromecast.socket_client.CONNECTION_STATUS_CONNECTED` to make it available when calling `entity.new_connection_status(connection_status)` below?